src/arena/Sprint.js

```diff
@@ -443,26 +466,32 @@ function Sprint(props) {
                   </section>{" "}
                 </div>
               </div>
-              <Accordion allowMultipleExpanded allowZeroExpanded>
+              <Accordion
```

Comment:
Ticket Item: Replace the Accordion npm package with styled semantic HTML. There are two cool element called `details` and `summary` that we should include across the codebase instead of Accordion Items. This will be better for screen readers and accessibility :) More info below:
https://dev.to/link2twenty/native-html-accordion-2fma

Reply:
Queued!